.github/workflows/web-api.yml

```diff
@@ -42,47 +108,24 @@ jobs:
           - 9200:9200
 
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          # fetch-depth is needed to get the full list of commits
-          # necessary for commit msg linting when a PR has 2 commits or more
-          # to avoid getting ALL the commits, we get current commits + origin
-          fetch-depth: ${{ github.event.pull_request.commits }} + 1
-          ref: ${{ github.event.pull_request.head.sha }}
-      - name: Setup python 3.6
+      - uses: actions/checkout@v2
+
+      - name: Setup Python
         uses: actions/setup-python@v2
         with:
           python-version: 3.6.9
-      - name: Checking commit quality (messages, signatures, etc)
-        env:
-          GIT_HEAD: ${{ github.event.after }}
-          GIT_ORIGIN: ${{ github.event.pull_request.base.sha }}
-        run: |
-          sh ./scripts/ci/prebuild.sh
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v2
-        with:
-          python-version: ${{ matrix.python-version }}
+
```

Comment:
We don't need these 2 steps here for `pytest`, that's why moved to a different one.

In case when commit quality fails, we don't proceed to python testing. IMO these are two different things and would be more visible if any one of them fails. wdyt?